postgresql/resource_postgresql_grant_test.go

```diff
@@ -550,6 +614,133 @@ resource "postgresql_grant" "test" {
 	})
 }
 
+func TestAccPostgresqlGrantForeignDataWrapper(t *testing.T) {
```

Comment:
The tests suite is run twice, the second time simulate an AWS RDS like environment with a non superuser environment, that's why the tests fail.

I spent some time trying to  make it works with the `postgres_fdw` extension as we can use it on RDS but it's not that simple.

For now, you can skip these tests for the second run, there's a `testSuperuserPreCheck` that helps for that (cf https://github.com/cyrilgdn/terraform-provider-postgresql/blob/master/postgresql/resource_postgresql_role_test.go#L84) but it will not work here as you create the `foreign data wrapper` before the `PreCheck`.

You can create a new helper based on `skipIfNotAcc` which behaves like `testSuperuserPreCheck`.